MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,593 @@
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs"%}
+dataGrid.Columns.Add( new DataGridTextColumn()
+{ 
+    MappingName = "OrderID"
+});
+{% endhighlight %}
+{% endtabs %}
+
+## DataGridCheckBoxColumn
+
+DataGridCheckBoxColumn inherits all the properties of `DataGridColumn`. It loads a switch as the content of record cells in the column and responds to value changes in it. The underlying data source can be changed that it toggles the values shown in the switch. The SfDataGrid automatically generates DataGridCheckBoxColumn if the property in the underlying collection of type set to bool.
+
+To use DataGridCheckBoxColumn, follow the code example:
+ 
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   ItemsSource="{Binding OrdersInfo}">
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridCheckBoxColumn MappingName="IsOnline" HeaderText="Is Online" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid> 
+{% endhighlight %}
+
+![DataGrid with checkbox column](Images\column-types\maui-datagrid-column-checkbox.png)
+
+
+## DataGridImageColumn
+
+DataGridImageColumn is derived from DataGridColumn. Hence, it inherits all the properties of DataGridColumn. It displays image as cell content of a column. To create DataGridImageColumn, the property corresponding to the column in the underlying collection must be `ImageSource` type.
+
+In DataGridImageColumn, it is possible to load images in any of the following four ways:
+
+ * **FromFile**: Required to specify the path of the file.
+ * **FromResource**: Required to set image as embedded resource.
+ * **FromStream**: Required to load image from byte[] array.
+ * **FromURI**: Required to set image from a web service or website.
+
+To load image (embedded resource) in DataGridImageColumn, follow the code example:
+ 
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="dataGrid" ItemsSource="{Binding OrderInfoCollection}"  ColumnWidthMode="Fill" utoGenerateColumnsMode="None">
+    <syncfusion:SfDataGrid.Columns >
+        <syncfusion:DataGridImageColumn  MappingName="DealerImage" HeaderText="Dealer Image" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+![DataGrid with image column](Images\column-types\maui-datagrid-column-imagecolumn.png)
+### Aspect
+
+SfDataGrid allows you to set the [Aspect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridImageColumn.html#Syncfusion_Maui_DataGrid_DataGridImageColumn_Aspect) to size the loaded images within the bounds of the grid cell (whether to stretch, crop or letterbox) using the `DataGridImageColumn.Aspect` property. The supported aspects are described below, the default value is AspectFit.
```

Comment:
@NirmalKumarYuvaraj  - Remove below sentence,

The supported aspects are described below, 

Also the below para,
* **AspectFill**: Clips the image so that it fills the display area while preserving the aspect (no distortion).

* **AspectFit**:  Letterboxes the image (if required) so that the entire image fits into the display area, with blank space added to the top/bottom or sides depending on whether the image is wide or tall.

* **Fill**: Stretches the image to completely and exactly fill the display area. This may result in the image being distorted.

* **Center**: Places the image to the center of the cell.

To set `Aspect` to images loaded inside `DataGridImageColumn`, refer the below code snippet.

Reply:
Replaced the content.